### PR TITLE
CIRC-1397 Disable request queue validation for page requests always being at the top

### PR DESCRIPTION
**Pull request by bisecomsergiy**
### Purpose
Change requests queue behavior to place the page requests at the queue top 
### Approach
Moved condition which allowed to place the page requests at the queue top.
### Learning
https://issues.folio.org/browse/CIRC-1397

### CODE REVIEW

File: src/main/java/org/folio/circulation/domain/validation/RequestQueueValidation.java [hunk 1]
```diff
@@ -87,24 +87,16 @@ private RequestQueueValidation() {
     }
 
     return result.value().isQueueForInstance()
-      ? pageRequestsHaveTopPositions(result)
-      : pageRequestHasFirstPosition(result);
+    ? result
+    : pageRequestHasFirstPosition(result);
```
Comment (alexanderkurash) [inline src/main/java/org/folio/circulation/domain/validation/RequestQueueValidation.java:91]: indentation

File: src/main/java/org/folio/circulation/domain/validation/RequestQueueValidation.java [hunk 2]
```diff
@@ -87,24 +87,16 @@ private RequestQueueValidation() {
...
   }
 
   /**
-   * Makes sure that all Page requests are always at the top of the unified queue (TLR feature
-   * enabled).
+   * Makes sure that all Page requests are always at the first position of the unified queue.
```
Comment (alexanderkurash) [inline src/main/java/org/folio/circulation/domain/validation/RequestQueueValidation.java:95]: Remove this comment. This method is not for unified queue, it's for old item-based request queues.

File: src/main/java/org/folio/circulation/domain/validation/RequestQueueValidation.java (hunk 1 above)
Comment (bisecomsergiy) [reply src/main/java/org/folio/circulation/domain/validation/RequestQueueValidation.java]: Refactored, thanks.

File: src/main/java/org/folio/circulation/domain/validation/RequestQueueValidation.java (hunk 2 above)
Comment (bisecomsergiy) [reply src/main/java/org/folio/circulation/domain/validation/RequestQueueValidation.java]: Comment is moved, thanks.

**Review by alexanderkurash — APPROVED** [review]

**Pull request merged**